py2many/cli.py

```diff
@@ -81,7 +81,10 @@ def core_transformers(tree, trees):
 
 
 def _transpile(
-    filenames: List[pathlib.Path], sources: List[str], settings: LanguageSettings
+    filenames: List[pathlib.Path],
+    sources: List[str],
+    settings: LanguageSettings,
+    _suppress_exceptions=Exception,
```

Comment:
This is a hidden arg, used only to support our self-transpile tests, and becomes unnecessary and can be removed once all languages can parse Python code without throwing exceptions.  We're getting quite close.  To me it is ridiculous to add typing here, especially as there are so many other typing problems to fix.

Reply:
It wasn't so much about internal/external or temporary/long term change. The most surprising thing about it was it could be a bool or a list of exceptions. It could be resolved with a comment, but type annotations work better sometimes.

Other mitigations are possible - split it into two variables or interpret empty list as boolean False etc.